.github/workflows/wasmcloud.yml

```
@@ -291,6 +291,7 @@ jobs:
           path: doc
 
   providers:
+    if: startswith(github.ref, 'refs/tags/provider-')
```

Hey @vados-cosmonic, I have seen the `startsWith` expression (or function) with lowercase `startswith`. The documentation is using `startsWith` pretty much everywhere. Since this is fairly not testable thing (other than eye-test). Should we rather go with? Did not found any sign that this would be case insensitive in the [docs](https://docs.github.com/en/actions/learn-github-actions/expressions#startswith).

```suggestion
    if: startsWith(github.ref, 'refs/tags/provider-')
```

---

Yikes, never mind it is all over the place here. 😬 🤦 At sorry for my triggered ptsd. At least these releases does not take hours.

---

Absolutely no problem @markkovari thanks for taking a close look -- yeah it *seems* like Github is OK with doing it either way, probably a ease-of-use feature. 

When I tested this on my own fork, it seemed to work fine. We could absolutely make it consistent across our codebase though, and avoid triggering more PTSD! :)

---

As long as you don't write "too much" bash in the steps and your visualization of the pipeline does not look like a circuit board I think you are good.